packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -77,6 +86,25 @@ export default class TelecomPackMigrationOffersCtrl {
     );
   }
 
+  static updateSelectedGtrOption(offer, optionName) {
+    Object.entries(offer.options).forEach(([, option]) => {
+      if (
+        option.name.startsWith('gtr_') &&
+        option.selected !== null &&
+        optionName.startsWith('gtr_')
+      ) {
+        if (option.name !== optionName) {
+          set(option, 'selected', false);
+        }
+      }
+    });
```

Comment:
**suggestion:** If the keys are not used, `Object.values` can be a better option
```suggestion
    Object.values(offer.options).forEach(([option) => {
      if (
        option.name.startsWith('gtr_') &&
        option.selected !== null &&
        optionName.startsWith('gtr_')
      ) {
        if (option.name !== optionName) {
          set(option, 'selected', false);
        }
      }
    });
```

Reply:
thx, done